src/lib/minions/functions/getUserBestGearFromBank.ts

```diff
@@ -0,0 +1,177 @@
+import { EquipmentSlot, Item } from 'oldschooljs/dist/meta/types';
+import { addBanks, removeItemFromBank } from 'oldschooljs/dist/util';
+
+import { GearTypes } from '../../gear';
+import { GearSetup, GearStat } from '../../gear/types';
+import { ItemBank } from '../../types';
+import { removeBankFromBank } from '../../util';
+import getOSItem from '../../util/getOSItem';
+
+function getItemScore(item: Item) {
+	return (
+		item?.equipment![GearStat.DefenceCrush] +
+		item?.equipment![GearStat.DefenceSlash] +
+		item?.equipment![GearStat.DefenceStab] +
+		item?.equipment![GearStat.DefenceMagic] +
+		item?.equipment![GearStat.DefenceRanged] +
+		item?.equipment![GearStat.AttackCrush] +
+		item?.equipment![GearStat.AttackSlash] +
+		item?.equipment![GearStat.AttackStab] +
+		item?.equipment![GearStat.AttackMagic] +
+		item?.equipment![GearStat.AttackRanged]
+	);
+}
+
```

Comment:
Do you need the `?` in `item?` - and maybe this would work more easily: `addArrayOfNumbers(Object.values(item.equipment))`